Updated What's New page 9/17/21 (#1889)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## September 6 - September 17, 2021

> - Enhanced List to support vertical alignment and added guidance for [multi-line lists](/templates/lists).

Reviewer: Enhanced List to support `top-alignment`.....

Author: Updated in latest commit

- Advanced Design System site search capabilities. Work in progress.

> - Aligned Toolbar Button hover implementation with [Figma](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164).

Reviewer 2: ```suggestion
- Aligned [Toolbar Button hover](/components/button#toolbar-button) implementation with [Figma](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164).
```

> - Completed research on [task focused navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18).

Reviewer 2: Trying to link to the relevant section of the Figma file.

```suggestion
- Completed research on [task focused navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1265%3A1187).
```

- Added guidance for [toggle icon buttons](/components/button#toggle-buttons-with-icons).

> - Completed a usability test on toggle buttons and sortable columns within a table utilizing a user who uses a screen reader.

Reviewer 2: ```suggestion
- Completed usability testing on toggle buttons and sortable columns within a table with an expert screen reader user.
```

> - Enhanced a11yText on our light/dark mode button on our [HPE Design System home page](https://design-system.hpe.design/).

Reviewer 2: Internal links should just use the path segment to avoid page reloads. I think '/' should work, but should be verified.

```suggestion
- Enhanced a11yText on our light/dark mode button within the HPE Design System [header navigation](/).
```

- [Ideation on Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1279%3A1206) coming to a finish.

> - Completed adding t-shirt sizing for buttons. Figma publishing coming soon!

Reviewer: Can we just say `Added t-shirt sizing for buttons. Figma publishing coming soon!

Author: Updated in latest commit
